src/components/Observations/map/ObservationMapView.tsx

```diff
@@ -1,23 +1,35 @@
 import { ObservationResults } from 'src/types/Observations';
-import { Box, useTheme } from '@mui/material';
+import { Box, Theme, useTheme } from '@mui/material';
 import MapLegend from 'src/components/common/MapLegend';
 import { PlantingSiteMap } from 'src/components/Map';
 import React, { useEffect, useMemo, useState } from 'react';
-import { MapEntityId, MapObject, MapSourceBaseData } from 'src/types/Map';
+import { MapEntityId, MapObject, MapSourceBaseData, MapSourceProperties } from 'src/types/Map';
 import { MapService } from 'src/services';
 import MapLayerSelect, { MapLayer } from 'src/components/common/MapLayerSelect';
 import strings from 'src/strings';
 import MapDateSelect from 'src/components/common/MapDateSelect';
 import { getRgbaFromHex } from 'src/utils/color';
 import { SearchInputProps } from 'src/components/Observations/search';
 import { regexMatch } from 'src/utils/search';
+import TooltipContents from 'src/components/Observations/map/TooltipContents';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  popover: {
+    '&.mapboxgl-popup': {
+      maxWidth: '324px !important', // !important to override a default mapbox style
+      width: '324px',
```

Comment:
why 324px? seems to have some extra space.. can this be dynamic based on content?

Reply:
That may be a good option. The widths are narrower in mobile, for example, in the designs.